src/apify_shared/consts.py

```diff
@@ -54,47 +54,60 @@ class ActorEventTypes(str, Enum):
     ABORTING = 'aborting'
 
 
+class ActorEnvVars(str, Enum):
+    """Possible Apify-specific environment variables prefixed with "ACTOR_"."""
+
+    # TODO: document these
+
+    #: BUILD_ID
+    BUILD_ID = 'ACTOR_BUILD_ID'
+    #: BUILD_NUMBER
+    BUILD_NUMBER = 'ACTOR_BUILD_NUMBER'
+    #: DEFAULT_DATASET_ID
+    DEFAULT_DATASET_ID = 'ACTOR_DEFAULT_DATASET_ID'
+    #: DEFAULT_KEY_VALUE_STORE_ID
+    DEFAULT_KEY_VALUE_STORE_ID = 'ACTOR_DEFAULT_KEY_VALUE_STORE_ID'
+    #: DEFAULT_REQUEST_QUEUE_ID
+    DEFAULT_REQUEST_QUEUE_ID = 'ACTOR_DEFAULT_REQUEST_QUEUE_ID'
+    #: EVENTS_WEBSOCKET_URL
+    EVENTS_WEBSOCKET_URL = 'ACTOR_EVENTS_WEBSOCKET_URL'
+    #: ID
+    ID = 'ACTOR_ID'
+    #: INPUT_KEY
+    INPUT_KEY = 'ACTOR_INPUT_KEY'
+    #: MAX_PAID_DATASET_ITEMS
+    MAX_PAID_DATASET_ITEMS = 'ACTOR_MAX_PAID_DATASET_ITEMS'
+    #: MEMORY_MBYTES
+    MEMORY_MBYTES = 'ACTOR_MEMORY_MBYTES'
+    #: RUN_ID
+    RUN_ID = 'ACTOR_RUN_ID'
+    #: STARTED_AT
+    STARTED_AT = 'ACTOR_STARTED_AT'
+    #: TASK_ID
+    TASK_ID = 'ACTOR_TASK_ID'
+    #: TIMEOUT_AT
+    TIMEOUT_AT = 'ACTOR_TIMEOUT_AT'
+    #: WEB_SERVER_PORT
+    WEB_SERVER_PORT = 'ACTOR_WEB_SERVER_PORT'
+    #: WEB_SERVER_URL
+    WEB_SERVER_URL = 'ACTOR_WEB_SERVER_URL'
+
+
 class ApifyEnvVars(str, Enum):
-    """Possible Apify-specific environment variables."""
+    """Possible Apify-specific environment variables prefixed with "APIFY_"."""
 
     # TODO: document these
```

Comment:
What is meant by this TODO? The class docstring is there, so maybe it could be removed?